numba/cuda/tests/cudadrv/test_nvvm_driver.py

```diff
@@ -25,6 +25,22 @@ def test_nvvm_compile_simple(self):
         self.assertTrue('simple' in ptx)
         self.assertTrue('ave' in ptx)
 
+    def test_nvvm_compile_nullary_option(self):
```

Comment:
Suggest adding a test for a "typo"ed option, like `nvvm.llvm_to_ptx(nvvmit, opt=3, this_is_not_an_option=1234)` and asserting that the error message produced is informative with respect to the mistake.

Reply:
I've added a test for this.